lpc11uxx/lpc11uxx.svd

```diff
@@ -2336,12 +2336,12 @@ v7 updates:
 							<enumeratedValues>
 								<name>ENUM</name>
 								<enumeratedValue>
-									<name>SEND_START_AND_STOP_</name>
+									<name>SEND</name>
```

Comment:
There are a lot of renames like this one which I'm not sure if they are that helpful :confused: this ends up with something like the following: `SSDIS::SEND` instead of `SSDIS::SEND_START_AND_STOP` which I consider *much* more readable.
I don't think everyone realizes immediately that `SSDIS` means "send start stop disable".

Reply:
Ok, you're right about this one (although we can omit the AND). Then the other variant would be `DO_NOT_SEND_START_STOP`, ok?

Reply:
Yes, sound alright :relaxed:

Reply:
Fixed